content/en/platform/corda/4.11/enterprise/node/collaborative-recovery/ledger-recovery/monitoring-ledger-recovery.md

```diff
@@ -0,0 +1,50 @@
+---
+date: '2023-11-08'
+menu:
+  corda-enterprise-4-11:
+    parent: corda-enterprise-4-11-ledger-recovery
+tags:
+- monitoring ledger recovery
+title: Monitoring Ledger Recovery process
+weight: 1000
+---
+
+# Monitoring Ledger Recovery process
+
+The Corda node log files provide initial informational messages and incremental updates on the number of records recovered at a given moment in time:
+
```

Comment:
I think these should be bullets like in the previous section?

Reply:
Yes, changed to bullets.